[2023-10-17T11:19:27.746+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-17T11:19:27.753+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-17T11:19:27.754+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:27.765+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-25 00:00:00+00:00
[2023-10-17T11:19:27.770+0000] {standard_task_runner.py:57} INFO - Started process 661 to run task
[2023-10-17T11:19:27.772+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-25T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpkmd5m2_5']
[2023-10-17T11:19:27.773+0000] {standard_task_runner.py:85} INFO - Job 119: Subtask waiting_for_data
[2023-10-17T11:19:27.807+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:27.862+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-25T00:00:00+00:00'
[2023-10-17T11:19:27.869+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:27.870+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:27.870+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:27.870+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:27.877+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210425T000000, start_date=20231017T111927, end_date=20231017T111927
[2023-10-17T11:19:27.904+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:27.920+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:00.473+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-18T10:43:00.483+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-18T10:43:00.483+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:00.516+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-25 00:00:00+00:00
[2023-10-18T10:43:00.523+0000] {standard_task_runner.py:57} INFO - Started process 1626 to run task
[2023-10-18T10:43:00.526+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-25T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpyjgbrheq']
[2023-10-18T10:43:00.527+0000] {standard_task_runner.py:85} INFO - Job 118: Subtask waiting_for_data
[2023-10-18T10:43:00.582+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:00.656+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-25T00:00:00+00:00'
[2023-10-18T10:43:00.674+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:00.677+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210425T000000, start_date=20231018T104300, end_date=20231018T104300
[2023-10-18T10:43:00.693+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 118 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1626)
[2023-10-18T10:43:00.742+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:00.773+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:36.762+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-18T11:17:36.774+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-18T11:17:36.774+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:36.789+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-25 00:00:00+00:00
[2023-10-18T11:17:36.796+0000] {standard_task_runner.py:57} INFO - Started process 510 to run task
[2023-10-18T11:17:36.799+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-25T00:00:00+00:00', '--job-id', '190', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpi213mmkk']
[2023-10-18T11:17:36.800+0000] {standard_task_runner.py:85} INFO - Job 190: Subtask waiting_for_data
[2023-10-18T11:17:36.860+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:36.972+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-25T00:00:00+00:00'
[2023-10-18T11:17:36.988+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:36.988+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:36.988+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:36.988+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:37.000+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210425T000000, start_date=20231018T111736, end_date=20231018T111737
[2023-10-18T11:17:37.057+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:37.120+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:42:55.566+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-19T12:42:55.580+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [queued]>
[2023-10-19T12:42:55.580+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:42:55.596+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-25 00:00:00+00:00
[2023-10-19T12:42:55.608+0000] {standard_task_runner.py:57} INFO - Started process 1073 to run task
[2023-10-19T12:42:55.614+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-25T00:00:00+00:00', '--job-id', '207', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp2gvnykfx']
[2023-10-19T12:42:55.615+0000] {standard_task_runner.py:85} INFO - Job 207: Subtask waiting_for_data
[2023-10-19T12:42:55.666+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-25T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:42:55.740+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-25T00:00:00+00:00'
[2023-10-19T12:42:55.750+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:42:55.751+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:42:55.751+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:42:55.751+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:42:55.760+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210425T000000, start_date=20231019T124255, end_date=20231019T124255
[2023-10-19T12:42:55.786+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:42:55.825+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
